README.md

```diff
@@ -6,6 +6,7 @@ List of the [GitHub](https://github.com) usernames of all the people interested
 - [martinagvilas](https://github.com/martinagvilas)
 - [galuhsahid](https://github.com/galuhsahid)
 - [Bhavani Ravi](https://github.com/bhavaniravi)
+
```

Comment:
This blank line is likely to divide the list in two. Can you remove it please?

Reply:
done. Sorry for the typo.